internal/backend/xattr_linux.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 Francisco Souza. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package backend
+
+import (
+	"github.com/pkg/xattr"
+)
+
+const XATTR_KEY = "user.metadata"
```

Comment:
nit: let's call it `xattrKey`, this way it's not exported (I know it's an internal package, but still heh).
```suggestion
const xattrKey = "user.metadata"
```

Reply:
Done.